lib/src/lifecycle_module.dart

```diff
@@ -103,12 +103,28 @@ abstract class LifecycleModule extends SimpleModule
       _didResumeController = new StreamController<LifecycleModule>.broadcast()
     ].forEach(manageStreamController);
 
-    _logger = new Logger('w_module');
+    _logger = new Logger('$name');
+    <
+        String,
+        Stream>{
+      'didLoad': didLoad,
+      'didLoadChildModule': didLoadChildModule,
+      'didResume': didResume,
+      'didSuspend': didSuspend,
+      'didUnload': didUnload,
+      'didUnloadChildModule': didUnloadChildModule,
+      'willLoad': willLoad,
+      'willLoadChildModule': willLoadChildModule,
+      'willResume': willResume,
+      'willSuspend': willSuspend,
+      'willUnload': willUnload,
+      'willUnloadChildModule': willUnloadChildModule,
+    }.forEach(_logLifecycleEvents);
   }
 
   /// Name of the module for identification in exceptions and debug messages.
   // ignore: unnecessary_getters_setters
-  String get name => _name;
+  String get name => _name ?? '$runtimeType';
```

Comment:
You are correct. If minification is applied `runtimeType` will return the minified symbol name.